web/src/components/storage/iscsi/InitiatorSection.tsx

```diff
@@ -24,17 +24,46 @@ import React from "react";
 
 import { _ } from "~/i18n";
 import { Page } from "~/components/core";
-import { InitiatorPresenter } from "~/components/storage/iscsi";
-import { useInitiator, useInitiatorChanges } from "~/queries/storage/iscsi";
+import {
+  DescriptionList,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  DescriptionListDescription,
+} from "@patternfly/react-core";
+import { InitiatorForm } from "~/components/storage/iscsi";
+import { useInitiator, useInitiatorMutation, useInitiatorChanges } from "~/queries/storage/iscsi";
+
+const InitiatorDescription = ({ initiator }) => {
+  return (
+    <DescriptionList aria-label={_("Initiator details")} isHorizontal>
```

Comment:
I wonder how it looks if using "isFluid" property.

Reply:
Better, indeed.